src/plugins/intel_cpu/src/node.h

```diff
@@ -618,8 +609,8 @@ class Node {
 
     std::string originalLayers;  // contains names of the original layers separated by comma
 
-    Node(const std::shared_ptr<ngraph::Node>& op, const dnnl::engine& eng, WeightsSharing::Ptr &w_cache, const ShapeInferFactory& shapeInferFactory);
-    Node(const std::string& type, const std::string& name, const dnnl::engine& eng, WeightsSharing::Ptr &w_cache);
+    Node(const std::shared_ptr<ngraph::Node>& op, RuntimeEnv::Ptr runtime_env, const ShapeInferFactory& shapeInferFactory);
+    Node(const std::string& type, const std::string& name, RuntimeEnv::Ptr runtime_env);
```

Comment:
Seems like Node should review const pointer as input.

Reply:
changed to const